schema.sql

```diff
@@ -0,0 +1,133 @@
+DROP
+DATABASE readme;
+
+CREATE
+DATABASE readme
+  DEFAULT CHARACTER SET utf8
+  DEFAULT COLLATE utf8_general_ci;
+
+USE
+readme;
+
+CREATE TABLE users
+(
+  id         INT AUTO_INCREMENT PRIMARY KEY,
+  registered DATETIME            NOT NULL,
+  email      VARCHAR(128) UNIQUE NOT NULL,
+  login      VARCHAR(128)        NOT NULL,
+  password   VARCHAR(64)         NOT NULL,
+  avatar     VARCHAR(125)
+);
+
+CREATE TABLE content_types
+(
+  id         INT AUTO_INCREMENT PRIMARY KEY,
+  name       VARCHAR(128) UNIQUE NOT NULL,
+  icon_class VARCHAR(32)
+);
+
+CREATE TABLE hashtags
+(
+  id   INT AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE posts
+(
+  id          INT AUTO_INCREMENT PRIMARY KEY,
+  created     DATETIME     NOT NULL,
+  title       VARCHAR(200) NOT NULL,
+  content     VARCHAR(1000),
+  author      VARCHAR(128),
+  picture     VARCHAR(125),
+  video       VARCHAR(125),
+  website     VARCHAR(125),
+  views_count INT DEFAULT 0,
```

Comment:
Для счетчика нам не нужна отрицательная часть инта, поэтому лучше сразу использовать UNSIGNED

```suggestion
  views_count INT UNSIGNED DEFAULT 0,
```